File: 프로그래머스/lv0/120837. 개미 군단/개미 군단.cs
using System;

public class Solution {
    public int solution(int hp) {
        
        int[] ATKs = new int[]{5, 3};
        
        int remainHp = hp;
        int antCount = 0;
        
        foreach(int atk in ATKs)
        {
            antCount += remainHp / atk; 
            remainHp %= atk; 
        }
        
        antCount += remainHp;
        
        return antCount;
    }
}

File: 프로그래머스/0/120837. 개미 군단/개미 군단.cpp
#include <string>
#include <vector>

using namespace std;

int solution(int hp) {
    int answer = 0;
    int arr[3] = { 5, 3, 1 }; 
    
    for (const int& atk : arr)
    {
        answer += hp / atk;
        hp %= atk;
    }
    
    return answer;
}